test/distributed/fsdp/test_fsdp_param_exec_order_policy.py

```diff
@@ -291,6 +308,38 @@ def test_train_with_full_state_dict(
             fsdp_model, fsdp_optim, ddp_model, ddp_optim, inp_shape, num_iters,
         )
 
+    @skip_if_lt_x_gpu(2)
+    def test_wrap_structure(self):
+        """
+        Tests that ``nn.Parameter`` are corrected grouped as ``FlatParameter``.
+        """
+        model = CNN().cuda()
+        group = dist.distributed_c10d._get_default_group()
+        # Used to test non-default module_level_group_policy
+        module_level_policy = functools.partial(
+            transformer_auto_wrap_policy,
+            transformer_layer_cls={DoubleConv, nn.Linear},
+        )
+        param_exec_order_policy = ParamExecOrderPolicy(
+                handle_init_mode=HandleInitMode.MODULE_LEVEL,
+                module_level_group_policy=module_level_policy,
+            )
+        fsdp_model = FSDP(model, group, auto_wrap_policy=param_exec_order_policy)
+        correct_prefixed_param_names = [
+            # here `shared.weight` should be in the same wrap as `root_weight`
```

Comment:
Add the explanation for why? Something like:
```suggestion
            # here `shared.weight` should be in the same wrap as `root_weight`
            # because the root module `model` is the lowest common ancestor of
            # `the modules conv1` and `conv2` that share `shared.weight`
```
(can rewrap/reformat as needed)